src/syntax/FStar.Syntax.Syntax.fsi

```diff
@@ -111,7 +111,7 @@ type term' =
   | Tm_arrow      of binders * comp                              (* (xi:ti) -> M t' wp *)
   | Tm_refine     of bv * term                                   (* x:t{phi} *)
   | Tm_app        of term * args                                 (* h tau_1 ... tau_n, args in order from left to right *)
-  | Tm_match      of term * list<branch>                         (* match e with b1 ... bn *)
+  | Tm_match      of term * option<ascription> * list<branch>    (* match e (ret asc?) with b1 ... bn *)
```

Comment:
Should this really be an `option<ascription>`? What would the option tactic there represent?